ui/components/Logo.tsx

```diff
@@ -15,10 +15,10 @@ type Props = {
 function Logo({ className }: Props) {
   return (
     <Spacer padding="medium">
-      <Flex className={className}>
+      <Flex className={className} align>
         <img src={logoSrc} />
         <Spacer padding="xxs" />
-        <img src={titleSrc} />
+        <img src={titleSrc} style={{ height: "90%" }} />
```

Comment:
What is this change trying to accomplish?

Reply:
Check it out without the set height - it stretches the text out to fit the container.